pkg/features/kube_features.go

```diff
@@ -36,6 +36,13 @@ const (
 	// of code conflicts because changes are more likely to be scattered
 	// across the file.
 
+	// owner: @ttakahashi21 @mkimuram
+	// kep: https://kep.k8s.io/3294
+	// alpha: v1.26
+	//
+	// Enable usage of Provision of PVCs from PVCs in other namespaces
+	CrossNamespaceSourceProvisioning featuregate.Feature = "CrossNamespaceSourceProvisioning"
```

Comment:
To be more consistent with the AnyVolumeDataSource feature gate, maybe a better name would be "CrossNamespaceVolumeDataSource"

Reply:
Please see https://github.com/kubernetes/kubernetes/pull/113186/commits/17f5ae6e3aaf7e970480ee5652d00f001f65efa4